linkcheck/parser/__init__.py

```diff
@@ -113,6 +113,9 @@ def parse_wml (url_data):
     find_links(url_data, url_data.add_url, linkparse.WmlTags)
 
 
+import threading
+parse_mutex = threading.Lock()
```

Comment:
this was extracted into #198 so it could have been merged without waiting for this one. I will rebase this one whenever that one is merged